playbooks/roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Hmm, I'm not sure if I understand your question. I think we'd run this role as part of building the image along with all the other roles. This role should also work in a running container too though. (Ideally, we'll never have to find out :smile: ).